ignite/engine/__init__.py

```diff
@@ -66,6 +72,8 @@ def create_supervised_evaluator(model, metrics={},
             with respect to the host. For other cases, this argument has no effect.
         prepare_batch (callable, optional): function that receives `batch`, `device`, `non_blocking` and outputs
             tuple of tensors `(batch_x, batch_y)`.
+        output_transform (callable, optional): function that receives 'x', 'y', 'y_pred' and returns value
+            to be assigned to engine's state.output after each iteration. Default is returning '(y_pred, y,)'
 
     Note: `engine.state.output` for this engine is a tuple of `(batch_pred, batch_y)`.
```

Comment:
Probably we need to remove this line

Reply:
Do you mean  line
Note: `engine.state.output` for this engine is a tuple of `(batch_pred, batch_y)`.
Yes. But better to update it.

Reply:
Yes, please update it